benchmarks/conftest.py

```diff
@@ -6,6 +8,21 @@ def seed(scope="session"):
     return 42
 
 
+def get_backend_id(param):
+    backend = param
+    return f"{backend=}"
+
+
+@pytest.fixture(params=[sparse._BACKEND], autouse=True, ids=get_backend_id)
```

Comment:
```suggestion
@pytest.fixture(params=[sparse._BACKEND.value], autouse=True, ids=get_backend_id)
```
So we get a simple string.

Reply:
Done.